firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -401,40 +207,29 @@ public void updateAppTask_whenAabReleaseAvailable_redirectsToPlay() throws Excep
     assertEquals(UpdateStatus.REDIRECTED_TO_PLAY, taskResult.getUpdateStatus());
   }
 
-  @Test
-  public void updateToLatestRelease_whenCalledMultipleTimes_cancelsPreviousTask() {
```

Comment:
I removed this because the mocked tasks for checkForUpdate/signInTester complete immediately. We'll need to add a countdownlatch or something similar to test this functionality. I've added a line to our build plan and so we can address this in a separate PR